bff/pkg/server/server.go

```diff
@@ -616,3 +614,94 @@ func ValidateDim(fl validator.FieldLevel) bool {
 	value := fl.Field().String()
 	return isValidDimension(value)
 }
+
+var validSystemTypes = []string{
+	"amazon_rds",
+	"google_cloudsql",
+	"azure_database",
+	"heroku",
+	"crunchy_bridge",
+	"aiven",
+	"tembo",
+	"self_hosted",
+}
+
+// A dbIdentifer is defined as <SystemID>-<SystemScope>-<SystemType>
+// SystemID: min 1 char maxx 63char
+// SystemScope:  <AWS_REGION>/<CLUSTER_PREFIX><AWS_ACCOUNT_ID> where AWS_REGION is between 1 and 50 characters, CLUSTER_PREFIX is between 0 and 11 characters, and AWS_ACCOUNT_ID is 12 characters. Then, the whole SystemScope is betwen 13 and 73 characters.
+// SystemType: One of the following values :
+// amazon_rds
+// google_cloudsql
+// azure_database
+// heroku
+// crunchy_bridge
+// aiven
+// tembo
+// self_hosted
```

Comment:
Updated the text.
```
// Here's the list of constraints for validating the inputs:
// - A dbIdentifer is defined as <SystemType>/<SystemID>/<SystemScope>
// - SystemID: min 1 char max 63char
// - SystemScope:  <AWS_REGION>/<CLUSTER_PREFIX><AWS_ACCOUNT_ID> where AWS_REGION is between 1 and 50 characters, CLUSTER_PREFIX is between 0 and 11 characters, and AWS_ACCOUNT_ID is 12 characters. Then, the whole SystemScope is betwen 13 and 73 characters.
// - SystemType: One of the following values :
//   - amazon_rds
//   - google_cloudsql
//   - azure_database
//   - heroku
//   - crunchy_bridge
//   - aiven
//   - tembo
//   - self_hosted

```

Reply:
addressed in d22763b3d771903a038cf1d31bfc39bc45dd2d1c